<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>544</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #2E2E2E; 
    color: #DDDDDD; 
    font-family: &quot;Segoe UI&quot;, &quot;SF Pro Text&quot;, Tahoma, Arial, sans-serif;
    font-size: 10pt;
}

QGroupBox {
    background-color: #383838; 
    border: 1px solid #444444;
    border-radius: 8px; 
    margin-top: 20px; 
    padding-top: 15px;
    color: #F0F0F0;
}

QGroupBox::title {
    font-weight: bold;
    font-size: 10pt;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 10px;
    margin-left: 5px;
    color: #FFFFFF;
}

QLineEdit, QSpinBox {
    background-color: #484848;
    border: 1px solid #505050;
    padding: 5px;
    border-radius: 5px;
}

QLineEdit {
    color: rgba(221, 221, 221, 0.7); 
}

QSpinBox {
	color: rgba(221, 221, 221, 0.7); 
}

QLineEdit:focus {
    border: 1px solid #4A90E2;
    color: #F0F0F0;
}

QListWidget {
    background-color: #3D3D3D;
    border: 1px solid #505050;
    border-radius: 5px;
    color: #DDDDDD;
    padding: 5px;
    outline: 0;
}

QListWidget::item:selected {
    background: #4A90E2; 
    color: #FFFFFF;
    border-radius: 3px;
}

QPushButton {
    background-color: #5C5C5C;
    border: none;
    color: #FFFFFF;
    padding: 8px 15px;
    border-radius: 6px;
    font-weight: 500;
}

QPushButton:hover {
    background-color: #6D6D6D;
}

QPushButton#generateButton, QPushButton#generateNickButton {
    background-color: #4A90E2; 
    font-weight: bold;
    color: #FFFFFF;
}

QPushButton#saveButton {
	background-color:  rgb(83, 201, 113); 
    font-weight: bold;
    color: #FFFFFF;
}

QPushButton#generateButton:hover, QPushButton#generateNickButton:hover {
    background-color: #5B9FEF;
}

QCheckBox {
    color: #DDDDDD;
    spacing: 5px;
}

QCheckBox::indicator {
    width: 14px;
    height: 14px;
    border-radius: 3px;
    border: 1px solid #777777;
    background-color: #505050;
}

QCheckBox::indicator:checked {
    background-color: #4A90E2; 
    border: 1px solid #4A90E2;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="generateTab" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>231</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 2px solid #202020;
    background-color: #383838;
}</string>
    </property>
    <widget class="QGroupBox" name="generatePass">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>221</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 2px solid #202020;
    background-color: #383838;
}</string>
     </property>
     <property name="title">
      <string>Password Configuration</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridPass">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <item row="3" column="0">
        <widget class="QCheckBox" name="cbDigits">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Digits</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="cbSymbols">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Symbols</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cbLower">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Lowercase</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="cbUpper">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Uppercase</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="generateButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>34</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#generateButton {
    background-color: #4A90E2;
    border: 2px solid #3A7BCD;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QSpinBox" name="lengthSpin">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLineEdit" name="passwordOutput">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="generateLogin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 2px solid #202020;
    background-color: #383838;
}</string>
     </property>
     <property name="title">
      <string>Identity Generator</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLogin">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="horizontalSpacing">
        <number>30</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLineEdit" name="loginInput">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cb_useInjection">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Symbol Injection</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="cb_useMushup">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Word Mashup</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="generateNickButton">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Generate ID</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="infoSave">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 2px solid #202020;
    background-color: #383838;
}</string>
     </property>
     <property name="title">
      <string>New Credential Entry</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
      <layout class="QGridLayout" name="gridInfo">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QLineEdit" name="siteInput">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="saveButton">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#saveButton {
    background-color: rgb(83, 201, 113);
    border: 2px solid rgb(99, 177, 124);
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="vaultTab" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>-10</y>
      <width>301</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <widget class="QGroupBox" name="manageGroup">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 2px solid #202020;
    background-color: #383838;
}</string>
     </property>
     <property name="title">
      <string>Database Management</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>30</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridMange" rowstretch="0" columnstretch="0,0">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="importButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#importButton {
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="exportButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#exportButton {
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="vaultGroup">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>251</width>
       <height>391</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 2px solid #202020;
    background-color: #383838;
}</string>
     </property>
     <property name="title">
      <string>Credentials Vault</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>31</y>
        <width>231</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridVault" rowminimumheight="0,0,0,0">
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="verticalSpacing">
        <number>30</number>
       </property>
       <item row="3" column="1">
        <widget class="QPushButton" name="deleteButton">
         <property name="styleSheet">
          <string notr="true">QPushButton#deleteButton {
    background-color: rgb(243, 122, 117);
    border: 2px solid rgb(191, 89, 99);
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLineEdit" name="searchInput">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QListWidget" name="vaultList">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget {
    background-color: #3D3D3D;
    border: 1px solid #505050;
    border-radius: 5px;
    color: #DDDDDD;
    padding: 5px;
    outline: 0; 
    alternate-background-color: #353535;
}

QListWidget::item:selected {
    background: #4A90E2; 
    color: #FFFFFF;
    border-radius: 3px;
}</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="viewDetailsButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#viewDetailsButton {
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>View Details</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
